be/src/olap/schema.cpp

```diff
@@ -147,4 +155,57 @@ vectorized::DataTypePtr Schema::get_data_type_ptr(FieldType type) {
     return nullptr;
 }
 
+vectorized::IColumn::MutablePtr Schema::get_predicate_column_nullable_ptr(FieldType type, bool is_null) {
+    vectorized::IColumn::MutablePtr ptr = Schema::get_predicate_column_ptr(type);
+    if (is_null) {
+        return doris::vectorized::ColumnNullable::create(std::move(ptr), doris::vectorized::ColumnUInt8::create());
+    }
+    return ptr;
+}
+
+vectorized::IColumn::MutablePtr Schema::get_predicate_column_ptr(FieldType type) {
+    switch (type) {
+        case OLAP_FIELD_TYPE_TINYINT:
+            return doris::vectorized::PredicateColumnType<doris::vectorized::Int8>::create();
```

Comment:
too much switch type case xxx, i have seen many times such functions, building a mapping may a good choice.
template technology may be useful to satisfied this need

Reply:
This method could just be called once,  branch doesn't matter.
And in the actual performance test, this won't cost too much time.